setup_cluster.sh

```diff
@@ -58,7 +58,7 @@ function validate_env() {
     exit 1
   fi
   if [[ -z ${SSH_PUBLIC_KEY} ]]; then
-    echo "Please set 'SSH_PUBLIC_KEY' environment variable."
+    echo "Environment variable 'SSH_PUBLIC_KEY' is not set. Please set it."
```

Comment:
to be consistent with this line: https://github.com/codeready-toolchain/toolchain-infra/pull/2/files#diff-a88c1d3771057755bf3be4f6e244165fR57